.github/workflows/ci.yml

```
@@ -2,20 +2,25 @@ name: CI
 
 on: [push, pull_request]
 
+env:
+  GIT_DEFAULT_BRANCH: ${{github.event.repository.default_branch}}
+
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-    name: CI
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: '2.7'
           bundler-cache: true
       - name: Lint RBI files
-        run: bundle exec rubocop
+        run: scripts/run_on_changed_rbis "bundle exec rubocop"
       - name: Typecheck RBI files
-        run: scripts/check_types
+        run: scripts/run_on_changed_rbis scripts/check_types
```

This looks wrong. I'm assuming it should be something like this:

```suggestion
        run: scripts/run_on_changed_rbis scripts "./check_types"
```

---

Ignore this, I misread